templates/artist/artist_list.html

```diff
@@ -13,9 +13,9 @@
     </div>
     <div class="filter-sort-options center">
         <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
-            <li><h5>Show me</h5></li>
```

Comment:
I think our CSS has an `h5` class that's not being used anymore and we can get rid of.